README.md

```diff
@@ -32,22 +30,18 @@ west update
 To build the application, run the following command:
 
 ```shell
-west build -b thingy91_nrf9160_ns asset_tracker_v2
+west build -b thingy91x/nrf9151/ns app
 ```
 
 If you are using the Powerfoyle solar shield, you can enable power monitoring
 capabilities by using this build command:
 
 ```shell
-west build -b thingy91_nrf9160_ns asset_tracker_v2 -- -DSHIELD=powerfoyle
+west build -b thingy91x/nrf9151/ns app -- -DSHIELD=powerfoyle
```

Comment:
i do not think this shield exists for thingy91 X?

Reply:
true, this shield only exists for the old thingy

Reply:
We planed to remove this in a separate PR. There is a boards folder that should be removed too as we do not intend to support that shield right?

Reply:
Removed